lib/src/json_table.cc

```diff
@@ -379,7 +379,7 @@ void TableReader::GetSchema(uintptr_t this_ptr, duckdb::ArrowSchemaWrapper& sche
     auto reader_copy = (*reader)->CloneShared();
 
     // Create arrow stream
-    auto stream_wrapper = duckdb::make_unique<duckdb::ArrowArrayStreamWrapper>();
+    auto stream_wrapper = std::make_unique<duckdb::ArrowArrayStreamWrapper>();
```

Comment:
I'm confused, this was changed from `duckdb::make_unique` to `std::make_unique`
instead of using `duckdb::make_uniq` ?